src/main/java/exception/BookAlreadyLostException.java

```diff
@@ -0,0 +1,7 @@
+package exception;
+
+public class BookAlreadyLostException extends Exception{
```

Comment:
여기만 일반 Excetpion인 이유가 있나요?
아울러 간단한 리뷰인데, 띄어쓰기도 잘 신경써주세요.

```suggestion
public class BookAlreadyLostException extends Exception {
```